applications/GeoMechanicsApplication/tests/cpp_tests/test_apply_constant_phreatic_multi_line_pressure_process.cpp

```diff
@@ -140,4 +142,45 @@ KRATOS_TEST_CASE_IN_SUITE(ApplyConstantPhreaticMultiLinePressureProcessDoesNotTh
         r_model_part, test_parameters))
 }
 
+KRATOS_TEST_CASE_IN_SUITE(ApplyConstantPhreaticMultiLinePressureProcessForShortDomain,
```

Comment:
If I understand the issue correctly, it occurs when some nodes of the domain are beyond the phreatic line, i.e. the phreatic line is 'too short'. To me, the name of the test does not really describe that situation. 
Maybe something like 
`ApplyConstantPhreaticMultilinePressureProcess_AppliesCorrectPressure_WhenPhreaticLineDoesNotSpanEntireDomain` would be more descriptive (it's quite long though, so I leave it up to you)

Reply:
Thank you for the suggestion. The title is long ;) but very descriptive.